### PR TITLE
Add Session support

### PR DESCRIPTION
**Pull request by BirmacherAkos**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -15,6 +15,8 @@ import (
 	"github.com/bitrise-io/go-utils/fileutil"
 	"github.com/bitrise-io/go-utils/log"
 	"github.com/bitrise-io/go-utils/pathutil"
+	"github.com/bitrise-io/steps-fastlane/session"
```
Comment (trapacska) [inline main.go]: We already have this package in the itunesconnect-deliver step.

File: main.go [hunk 2]
```diff
@@ -139,6 +141,26 @@ func main() {
 
 	log.Donef("Expanded WorkDir: %s", workDir)
 
+	//
+	// Fastlane session
+	fmt.Println()
+	log.Infof("Ensure cookies for Apple Developer Portal")
+
+	fs, err := session.GetSession()
```
Comment (trapacska) [inline main.go]: Same comments as on the other PRs.

File: session/session.go [hunk 3]
```diff
@@ -0,0 +1,144 @@
+package session
```
Comment (trapacska) [inline session/session.go]: the only(mostly) reason we use packages to use them somewhere else, please do not duplicate the code base, use the pacage from the other PR.

**Review by trapacska — CHANGES_REQUESTED** [review]

File: session/session.go (hunk 3 above)
Comment (godrei) [inline session/session.go]: import this functionality from the deliver step

**Review by godrei — CHANGES_REQUESTED** [review]
Pls have a look at my notes

**Review by trapacska — APPROVED** [review]

**Review by lszucs — APPROVED** [review]

**Pull request merged**